### ISSUE TITLE
Error when downloading fansly content from coomer

### ISSUE THREAD

**Issue by ZapperDJ** [darkdragn/party#28]
I tried to download the archive dfor different creators tha used a fansly account and get this error everytime:

```
Traceback (most recent call last):

  File "/home/user/.local/bin/party", line 8, in <module>
    sys.exit(APP())

  File "/home/user/.local/lib/python3.10/site-packages/party/cli.py", line 361, in search
    pull_user(

  File "/home/user/.local/lib/python3.10/site-packages/party/cli.py", line 172, in pull_user
    posts = list(

  File "/home/user/.local/lib/python3.10/site-packages/party/user.py", line 112, in generate_posts
    yield schema.load(post)

  File "/home/user/.local/lib/python3.10/site-packages/marshmallow/schema.py", line 722, in load
    return self._do_load(

  File "/home/user/.local/lib/python3.10/site-packages/marshmallow/schema.py", line 909, in _do_load
    raise exc

marshmallow.exceptions.ValidationError: {'id': ['Not a valid integer.']}
```

**Comment by darkdragn** [comment]
I just checked, actually not where I thought and not a regression, for some fansly posts it's returning a weird ID. For example:

```
{'id': 'DM370373687712423936370373687527878656'
```

I'll patch it, but I need to op test against kemono and onlyfans stuff. Shouldn't take long

**Comment by darkdragn** [comment]
The fix is in v0.7.3. Give it a run and let me know how it works out for you!

**Comment by ZapperDJ** [comment]
Tried it now and it is downloading, seems like the problem is gone :)
